tests/testCModel.py

```diff
@@ -43,7 +43,7 @@ def makeMultiShapeletCircularGaussian(sigma):
     s = lsst.shapelet.ShapeletFunction(0, lsst.shapelet.HERMITE, sigma)
     s.getCoefficients()[0] = 1.0 / lsst.shapelet.ShapeletFunction.FLUX_FACTOR
     m = lsst.shapelet.MultiShapeletFunction()
-    m.getComponents().push_back(s)
+    m.addComponent(s)
```

Comment:
This is adapting to a necessary change made by @pschella when wrapping the `shapelet` package.  This object previously exposed a non-const reference to a STL container that needed to be modified in place.  That's no longer possible now that STL containers are only wrapped as copies.